### DISCUSSION TITLE
Confirming 100% MTRs for single parent on legacy benefits with no hours worked

### DISCUSSION THREAD

**Discussion by MaxGhenis** [PolicyEngine/policyengine#142] [Q&A] [answered]
Here's what I entered:
* Age=32
* Claims legacy benefits
* One child

![image](https://user-images.githubusercontent.com/6076111/140964324-6c557044-af9c-419b-8705-933d32463903.png)

When I add working hours, the MTR only goes to 73%:
![image](https://user-images.githubusercontent.com/6076111/140964713-893fce09-e79d-4909-a477-e146ac8ebd21.png)

Which program produces 100% MTRs?

**Comment by nikhilwoodruff** [comment, marked as answer]
At a guess, it looks like Income Support to me. That has a maximum hours limit for eligibility, so I think the 73% is caused by losing Income Support (100% MTR) and switching to the Tax Credits (41% + 20% + 12% = 73%).

**Reply by MaxGhenis** [reply]
Ah yep, I didn't see a phase-out rate parameter for Income Support, but the formula indicates that it's 100%:
https://github.com/PolicyEngine/openfisca-uk/blob/1f2a48cba55b4620ade6c6ee8267aaf30e08429f/openfisca_uk/variables/finance/benefit/family/income_support.py#L160-L163

Thanks!
